core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -495,6 +496,9 @@ void core_init_mmu_regs(void)
 {
 	uint32_t ttbcr = TTBCR_EAE;
 	uint32_t mair;
+	paddr_t ttbr0;
+
+	ttbr0 = virt_to_phys((void *)l1_xlation_table[get_core_pos()]);
```

Comment:
Please be consistent with how you do this in the two versions of `core_init_mmu_regs()` in this file.